server/datastore/mysql/migrations/tables/20240424124712_AddSoftwareInstallerTables.go

```diff
@@ -110,6 +110,9 @@ CREATE TABLE IF NOT EXISTS host_software_installs (
   -- Exit code of the post-script run after the software is installed
   post_install_script_exit_code int(10) DEFAULT NULL,
 
+  -- User that requested the installation, for upcoming activities
+  user_id int(10) unsigned DEFAULT NULL,
```

Comment:
Heads-up! This will require `make db-reset` on your local environments to re-apply the DB migration.

Reply:
@ghernandez345 @roperzh @gillespi314 @dantecatalfamo @jahzielv for visibility ^ (db migration updated, will require db-reset in your local envs).